examples/requirements.txt

```diff
@@ -0,0 +1,54 @@
+amqp==1.4.9
```

Comment:
TODO because of https://github.com/inveniosoftware/invenio-records-rest/pull/103#discussion_r71332337